protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -115,16 +115,13 @@ service TradingDataService {
   // List ledger entries
   //
   // Get a list of ledger entries within the given date range.
-  // This query requests and sums the number of ledger entries from a given subset of accounts, specified via the 'filter' argument.
-  // It returns a time series - implemented as a list of AggregateLedgerEntry structs - with a row for every time
-  // the summed ledger entries of the set of specified accounts changes.
-  // Each account filter must contain no more than one party ID.
+  // This query returns all the ledger entries for a given subset of accounts, specified via the 'filter' argument.
```

Comment:
I don't think we need this line at all really because right below we say how entries can be filtered, and I think its obvious that the field called `filter` is used to filter the results.

```suggestion
```